decoders/connector/lansitec/badge-tracker/v1.0.0/payload.js

```diff
@@ -1,609 +1,188 @@
-function float16(bytes) {
-  console.log(bytes.toString("hex"));
-  const sign = bytes[0] >> 7 === 1? -1 : 1;
-  const bias = 0x0f;
-  const exponent = (bytes[0] & 0x7c) >> 2;
-  const fraction = (bytes[0] & 0x03) << 8 | bytes[1];
+// Badge Tracker

```

Comment:
This decoder must have a unit test, here is an example of a unit test you can use.
```ts
import { describe, test, expect } from "vitest";

import { decoderRun } from "../../../../../src/functions/decoder-run";

const file_path = "decoders/connector/your-path/payload.js" as const;

describe("Shall not be parsed", () => {
  let payload = [{ variable: "shallnotpass", value: "04096113950292" }];
  payload = decoderRun(file_path, { payload });
  test("Output Result", () => {
    expect(Array.isArray(payload)).toBe(true);
  });

  test("Not parsed Result", () => {
    expect(payload).toEqual([{ variable: "shallnotpass", value: "04096113950292" }]);
  });
});
```

If you can add a unit test for the payload parser, that would be great.